daft/dataframe/dataframe.py

```diff
@@ -158,6 +158,28 @@ def _result(self) -> Optional[PartitionSet]:
         else:
             return self._result_cache.value
 
+    def explain_broadcast(self):
```

Comment:
Private method -- not intended for users to call

Reply:
Updated to have an underscore in front of name (i.e., `_explain_broadcast`).